setup.py

```diff
@@ -415,5 +415,6 @@ def run(self):
     },
 
     # expand the project metadata dictionary to fill in those values
-    **PROJECT_METADATA
+    **PROJECT_METADATA,
+    license_file='LICENSE.txt'
```

Comment:
Is this correct? I tried finding this key in the setup.py docs, but only found it in setup.cfg

Reply:
It is based on https://setuptools.pypa.io/en/latest/references/keywords.html. But I just noticed that the docs says it's deprecated and we should instead use `license_files` which defaults to `LICENSE*`.